src/Redemption.js

```diff
@@ -275,7 +275,7 @@ export default class Redemption {
         // However, constructFundingProof includes it for deposit funding
         // proofs. Here, we filter it out to produce the right set of
         // parameters.
-        ...proofArgs.filter(_ => _ != "output position")
+        ...proofArgs.filter(_ => _ != -1)
```

Comment:
Again, I know this is legacy, but do we want to prefer `!==` instead of `!=` in the spirit of stricter typing?

Reply:
Yah can do.

Reply:
Fix't.